coordinator/src/services/handlers/spine-handler.ts

```diff
@@ -22,20 +22,20 @@ const httpsAgent = new https.Agent({
 export class LiveRequestHandler implements RequestHandler {
   private readonly spineEndpoint: string
   private readonly spinePath: string
-  private readonly ebXMLBuilder: (spineRequest: SpineRequest) => string
+  private readonly ebXMLBuilder: (spineRequest: SpineRequest, xRequestIdHeader: string) => string
 
   constructor(
     spineEndpoint: string = null,
     spinePath: string = null,
-    ebXMLBuilder: (spineRequest: SpineRequest) => string = null
+    ebXMLBuilder: (spineRequest: SpineRequest, xRequestIdHeader: string) => string = null
```

Comment:
Could we update the definition of spine request to also include a field for spine request so that we don't need to independently pass around the request ID?